eng/common/scripts/Detect-Api-Changes.ps1

```diff
@@ -0,0 +1,105 @@
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [string] $ArtifactPath,
+  [Parameter(Mandatory=$True)]
+  [string] $PullRequestNumber,
+  [Parameter(Mandatory=$True)]
+  [string] $BuildId,
+  [Parameter(Mandatory=$True)]
+  [string] $CommitSha,
+  [Parameter(Mandatory=$True)]
+  [string] $ArtifactList,
```

Comment:
If you take my suggestion above this will need to become an array instead of just a string.

Reply:
Updated to array as per above suggestion.